contact.css

```diff
@@ -0,0 +1,253 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+* {
+  font-family: Poppins, sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+section {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background: #112d42;
+}
+section::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 50%;
+  min-height: 100vh;
+  background: aqua;
+}
+section .container {
+  position: relative;
+  min-width: 1100px;
+  min-height: 550px;
+  z-index: 1000;
+  display: flex;
+}
+section .container .contactinfo {
+  position: absolute;
+  top: 40px;
+  width: 350px;
+  height: calc(100% - 80px);
+  background: #0f3959;
+  z-index: 1;
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-shadow: 0px 20px 20px rgb(0, 0, 0, 0.2);
+}
+section .container .contactinfo h2 {
+  color: white;
+  font-size: 24px;
+  font-weight: 500;
+}
+section .container .contactinfo .info {
+  position: relative;
+  margin: 20px 0;
+}
+section .container .contactinfo .info li {
+  position: relative;
+  list-style: none;
+  display: flex;
+  align-items: flex-start;
+  cursor: pointer;
+  margin: 20px 0;
+}
+
+section .container .contactinfo .info li span:nth-child(1) {
+  width: 30px;
+  min-width: 30px;
+}
+section .container .contactinfo .info li span:nth-child(1) img {
+  width: 100%;
+  filter: invert(1);
+  opacity: 0.5;
+}
+section .container .contactinfo .info li span:nth-child(2) {
+  font-weight: 300;
+  color: white;
+  opacity: 0.5;
+  margin-left: 10px;
+}
+section .container .contactinfo .info li:hover span:nth-child(1) img,
+section .container .contactinfo .info li:hover span:nth-child(2) {
+  opacity: 1;
+}
+section .container .contactinfo .sci {
+  display: flex;
+  position: relative;
+}
```

Comment:
```suggestion
align-self: center;
}
```
![image](https://user-images.githubusercontent.com/91013960/198888844-242ca502-b7bd-4f80-9af4-2e05e6876007.png)
![image](https://user-images.githubusercontent.com/91013960/198888902-48fd5752-4f32-4e78-ba37-7cb3a2ad9f10.png)

this will center your ul tag and it looks nice in mobile view and also desktop view see above images